inventory/sample/group_vars/etcd.yml

```diff
@@ -18,6 +18,10 @@
 # This affects ETCD_PEER_CLIENT_CERT_AUTH variable
 # etcd_peer_client_auth: true
 
+## Container runtime
+## docker for docker, crio for cri-o and containerd for containerd.
+# container_manager: containerd
```

Comment:
There is already in sample inventory as https://github.com/kubernetes-sigs/kubespray/blob/da8522af6456758b135257f3ba2bbfb49d8ffc1e/inventory/sample/group_vars/k8s_cluster/k8s-cluster.yml#L203-L206

Reply:
The purpose I guess is to allow to have a different container_manager on `etcd` nodes? Something like `docker` on dedicated `etcd` nodes and `containerd` for kubernetes nodes.

Is this your intention @vi7 ?

Reply:
Hi @oomichi , @cristicalin ! thanks for reviewing this. What I meant is that [this](https://github.com/kubernetes-sigs/kubespray/blob/master/roles/kubernetes/preinstall/tasks/0020-verify-settings.yml#L288) check is failing when I have `etcd_deployment_type: docker` in the [etcd.yml](https://github.com/kubernetes-sigs/kubespray/blob/master/inventory/sample/group_vars/etcd.yml#L23) and `container_manager: docker` in the [k8s-cluster.yml](https://github.com/kubernetes-sigs/kubespray/blob/master/inventory/sample/group_vars/k8s_cluster/k8s-cluster.yml#L206) while it must not

The reason is that mentioned check happens only on the hosts from group `etcd` while what @oomichi showed is related to the `k8s-cluster` _only_, hence `etcd` hosts are inheriting container_manager from the defaults which is not ok.

And as a bonus feature it indeed allows to properly configure different container_manager for `etcd` hosts as @cristicalin  mentioned.

Reply:
Thanks for your explanation @vi7 
I got the point.
My small concern is this additional lines can be accidentally deleted because the lines are completely same and it is a little hard to understand the reason why we need to maintain the same lines in the inventory.
It is nice to write the reason to make those lines different like
```
## Container runtime for etcd nodes
## docker for docker, crio for cri-o and containerd for containerd.
## It is possible to specify different container_manager on etcd nodes.
```
Or something.

Reply:
@oomichi done, pls check if extended comment message seems fine for you